src/main/java/io/github/syst3ms/skriptparser/tags/TagReset.java

```diff
@@ -24,7 +25,7 @@ public boolean init(String key, String[] parameters) {
 	}
 
 	public String getValue(String affected) {
-		return affected;
+		return ConsoleColors.RESET + affected;
```

Comment:
While this implementation is correct for now, it reminds me that a small overhaul of the tag system is needed in order to give addons the flexibility of changing the reset tag.